cardano-api/src/Cardano/Api/Eras.hs

```diff
@@ -239,7 +241,8 @@ deriving instance Show (ShelleyBasedEra era)
 -- of Shelley-based eras, but also non-uniform by making case distinctions on
 -- the 'ShelleyBasedEra' constructors.
 --
-class IsCardanoEra era => IsShelleyBasedEra era where
+class (IsCardanoEra era, Ledger.Crypto (ShelleyLedgerEra era) ~ StandardCrypto)
```

Comment:
I needed this `Ledger.Crypto (ShelleyLedgerEra era) ~ StandardCrypto` constraint in `Cardano.CLI.Shelley.Run.Query`.